pkg/crc/services/dns/dns_darwin.go

```diff
@@ -131,7 +131,7 @@ func updateResolvConfFile(instanceIP string, resolvConfFile string) (bool, error
 
 	for i, ns := range hostResolv.NameServers {
 		// Update the nameserver IP with instance IP if already exist
-		if strings.Contains(ns.IPAddress, "192.168.130") {
+		if strings.Contains(ns.IPAddress, "192.168.130") || strings.Contains(ns.IPAddress, "192.168.64") {
```

Comment:
can we be more specific about the IP address range and create constants to refer to?

Reply:
@gbraad will be obsolete by #315